openedx/core/djangoapps/content/search/tests/test_documents.py

```diff
@@ -268,7 +267,6 @@ def test_html_library_block(self):
         doc = {}
         doc.update(searchable_doc_for_library_block(self.library_block))
         doc.update(searchable_doc_tags(self.library_block.usage_key))
-        doc.update(searchable_doc_collections(self.library_block.usage_key))
```

Comment:
@navinkarkera what is the reason for deleting this?

Reply:
Not needed any more.